.github/workflows/ci-pipeline.yaml

```
@@ -63,136 +35,67 @@ jobs:
-  auto-pr-review:
+          VIB_ENV_CONTAINER: ${{ steps.get-container-metadata.outputs.name  }}
+          VIB_ENV_TAG: ${{ steps.get-container-metadata.outputs.tag }}
+  verify-all:
+    # Ensure all containers passed the verification
     runs-on: ubuntu-latest
+    name: Verify
     needs: vib-verify
-    name: Reviewal for automated PRs
-    if: |
-      always() &&
-      github.actor == 'bitnami-bot' &&
-      contains(github.event.pull_request.labels.*.name, 'auto-merge')
+    if: ${{ always() }}
     steps:
-      # Approves the CI's PR if the 'VIB Verify' job succeeded
-      # Approved by the 'github-actions' user. A PR can't be approved by its author
-      - name: Approval
-        if: ${{ needs.vib-verify.result == 'success' }}
-        run: |
-          curl --request POST \
-          --url https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews \
-          --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
-          --header 'content-type: application/json' \
-          --data '{
-            "event": "APPROVE"
-            }' \
-          --fail
-      - name: Remove auto-merge label
-        if: ${{ needs.vib-verify.result == 'failure' }}
-        uses: andymckay/labeler@1.0.4
-        with:
-          repo-token: "${{ secrets.GITHUB_TOKEN }}"
-          remove-labels: "auto-merge"
-      - name: Disable auto-merge
-        if: ${{ needs.vib-verify.result == 'failure' }}
-        run: |
-          curl --request POST \
-          --url https://api.github.com/graphql \
-          --header 'authorization: Bearer ${{ secrets.BITNAMI_BOT_TOKEN }}' \
-          --data '{
-            "query": "mutation { disablePullRequestAutoMerge(input: {pullRequestId: \"${{ github.event.pull_request.node_id }}\"}) { clientMutationId }}"
-            }' \
-          --fail
-      - name: Manual review required
-        if: ${{ needs.vib-verify.result == 'failure' }}
-        run: |
-          curl --request POST \
-          --url https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.pull_request.number }}/comments \
-          --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
-          --header 'content-type: application/json' \
-          --data '{
-            "body": "There has been an error during the automated release process. Manual revision is now required."
-            }' \
-          --fail
-          curl --request POST \
-          --url https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/requested_reviewers \
-          --header 'authorization: Bearer ${{ secrets.BITNAMI_BOT_TOKEN }}' \
-          --header 'content-type: application/json' \
-          --data '{
-            "reviewers": ["fmulero"]
-            }' \
-          --fail
+      - name: Verify
+        if: ${{ needs.vib-verify.result != 'success' && needs.vib-verify.result != 'skipped'}}
```

Can `needs.vib-verify.result` status be different from `success/skipped/failure`? If not, we may simplify this condition to:

 ```suggestion
        if: ${{ needs.vib-verify.result == 'failure'}}
```

---

`success/skipped/failure/cancelled` -> https://docs.github.com/en/actions/learn-github-actions/contexts#needs-context

---

My bad, I forgot about the `canceled` one introduced by the matrix logic. 😄